### PR TITLE
Add form for signing up to email updates

### LINKED ISSUE #217: Allow non-users to subscribe to updates
We should have a form to allow non-logged-in users to submit their email address.

### PR DESCRIPTION
**Pull request by danlamanna**
Fixes #217

### CODE REVIEW

File: stade/core/templates/base.html [hunk 1]
```diff
@@ -99,6 +99,26 @@ <h1><a href="{% url 'index' %}"><img src="{% static 'img/isic_logo.svg' %}" alt=
     {% load invites %}
 
     {% show_pending_invites request %}
+
+    {% if not user.is_authenticated %}
+      <div class="has-text-centered">
```
Comment (brianhelba) [inline stade/core/templates/base.html:104]: This is pretty intrusive at this location at the top of every page. Can we put it in the menu bar maybe?

File: stade/core/templates/base.html [hunk 2]
```diff
@@ -131,5 +151,41 @@ <h1><a href="{% url 'index' %}"><img src="{% static 'img/isic_logo.svg' %}" alt=
     <div class="help-container">
       <a class="help-button button is-medium is-warning" href="mailto:support@isic-archive.com" target="_blank">Report an Issue</a>
     </div>
+
+    <script type="text/javascript">
```
Comment (brianhelba) [inline stade/core/templates/base.html:155]: Would it be possible to include this JS and the HTML that it interacts with in a separate template, then `{% include ` it from `base.html`? I think that'll make it easier to maintain, and keep the source of `base.html` from becoming too massive.

File: stade/core/templates/base.html (hunk 1 above)
Comment (danlamanna) [reply stade/core/templates/base.html:104]: I'd rather add an x sign and save that in local storage.

**Comment by brianhelba** [comment]
This should again merge cleanly.

**Review by brianhelba — CHANGES_REQUESTED** [review]
I'm a little thrown off by its location. Do you think it's a good UX?

**Comment by brianhelba** [comment]
@danlamanna Back to you to style in a way compatible with the new banner.

**Comment by jtomeck** [comment]
@danlamanna Sorry it took me a while to get to this, the interviews and some other projects messed with my schedule. I've moved the subscribe form up into the header as a toggleable dropdown.

**Pull request merged**